app-crypt/zulucrypt/zulucrypt-5.4.0_pre20180223.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Front end to cryptsetup"
+HOMEPAGE="https://mhogomchungu.github.io/zuluCrypt/"
+MY_COMMIT="76637bb05af13744bf1734b56f67d6d5cc2343b1"
+SRC_URI="https://github.com/mhogomchungu/zuluCrypt/archive/${MY_COMMIT}.zip -> ${P}.zip"
```

Comment:
Please make use of `vcs-snapshot.eclass` here. It will also take care of `S` directory.

Reply:
Fixed.